storage/buckets/main.go

```diff
@@ -208,15 +213,27 @@ func removeUser(c *storage.Client, bucketName string) error {
 	ctx, cancel := context.WithTimeout(ctx, time.Second*10)
 	defer cancel()
 	bucket := c.Bucket(bucketName)
-	policy, err := bucket.IAM().Policy(ctx)
+	policy, err := bucket.IAM().V3().Policy(ctx)
 	if err != nil {
 		return err
 	}
 	// Other valid prefixes are "serviceAccount:", "user:"
 	// See the documentation for more values.
 	// https://cloud.google.com/storage/docs/access-control/iam
-	policy.Remove("group:cloud-logs@google.com", "roles/storage.objectViewer")
```

Comment:
We should probably make these arguments to the sample, WDYT? @frankyn

Reply:
We generally try to have as few arguments for sample functions. So, if it's not needed for testing, generally prefer to not make it an argument.

https://github.com/GoogleCloudPlatform/golang-samples/blob/master/CONTRIBUTING.md#function-arguments-for-snippets

Reply:
It's not necessary, but existing samples introduce role, member as parameters.

https://cloud.google.com/storage/docs/access-control/using-iam-permissions

If it's a hard requirement not to then that's okay, but would be more consistent for GCS samples.

Reply:
👍

Reply:
Consistent within the product sounds good to me. Nice when the samples on a single page are consistent. Whatever y'all think. :)